fundamentals/middleware/rate-limit/WebRateLimitAuth/Program.cs

```diff
@@ -168,7 +168,6 @@
 #elif FIRST
 // <snippet_1>
 // <snippet>
-// Preceding code removed for brevity.
```

Comment:
In the second commit I removed this line because I found it confusing, because the docs show the entire file. https://learn.microsoft.com/en-us/aspnet/core/performance/rate-limit?view=aspnetcore-9.0#limiter-with-onrejected-retryafter-and-globallimiter

I can revert it if needed.